Mage/src/main/java/mage/abilities/dynamicvalue/common/CountersSourceCount.java

```diff
@@ -61,4 +62,18 @@ public String toString() {
     public String getMessage() {
         return (counterType != null ? counterType.toString() + ' ' : "") + "counter on {this}";
     }
+
+    @Override
+    public String getMessage(ValuePhrasing phrasing) {
+        String optCounterName = counterType != null ? counterType.toString() + ' ' : "";
+        // TODO: add option to use "it" instead of {this}
```

Comment:
do you have examples where that is needed?

(there is some auto-replacement logic but probably not applicable here)

Reply:
[[Rusting Golem]] came up in the oracle text errors diff (not a new error, just different)
```
Wrong card 27405: Rusting Golem
 + Fading 5
-- Rusting Golem's power and toughness are each equal to the number of fade counter on Rusting Golem.
+- Rusting Golem's power and toughness are each equal to the number of fade counters on Rusting Golem.
 ref:
 + Fading 5
 - Rusting Golem's power and toughness are each equal to the number of fade counters on it.
```

Contrast with [[Hancock, Ghoulish Mayor]]